lib/index.js

```diff
@@ -216,7 +216,7 @@ module.exports = foldl(function transform(ret, pairs, method) {
       var flattened = [].concat.apply([], map(function(words) {
         return words.split(' ');
       }, permutation));
-      return '^[ _]?' + flattened.join('[ _]?') + '[ _]?';
+      return '^[ _]?' + flattened.join('[ _]?') + '[ _]?$';
```

Comment:
😱 Only the last element in the regex had an end of line due to `var conjoined = values.join('|') + '$';`

We can remove the extra one there now.